dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/query/DefaultQueryParser.java

```diff
@@ -72,7 +72,14 @@ public Query parse( Class<?> klass, List<String> filters, Junction.Type rootJunc
             if ( split.length >= 3 )
             {
                 int index = split[0].length() + ":".length() + split[1].length() + ":".length();
-                query.add( getRestriction( schema, split[0], split[1], filter.substring( index ) ) );
+
+                if ( split[0].equals( "identifiable" ) )
```

Comment:
Should we check for schema.hasPErsistedProp here? I know there isn't much of a chance.. but if we have a property called identifiable on a new type, it would be hidden away...